lib/widgets/team_member_widget.dart

```diff
@@ -1,18 +1,15 @@
 import 'package:flutter/material.dart';
 
-class TeamMember extends StatefulWidget {
-  final double width;
-  final double height;
```

Comment:
We don't need hardcoded sizes here anymore. We will just use the size that we get from the circular list